packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -97,7 +97,6 @@
     cursor: pointer;
     display: block;
     font-size: var(--terra-consumer-profile-link-font-size, 14px);
-    margin-top: 3px;
```

Comment:
it seems this removes the punch-through in mobile

Reply:
i tested setting 
```
--terra-consumer-profile-link-border-top: 3px solid #9BDAF1
```
which is what we have in dex. and it shows up fine
![image](https://user-images.githubusercontent.com/8029578/31344332-fc23d9fc-acd7-11e7-9b1d-6e68b9167c48.png)

Reply:
If the border isnt an explicit color it removes it yes. I can try to find a fix to that